File: backend/src/main/java/com/plugga/backend/TestController.java
package com.plugga.backend;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.net.InetAddress;
import java.net.UnknownHostException;

@RestController
public final class TestController {

    @GetMapping
    public final String test() throws UnknownHostException {
        return "Hello from TestController!!!! My address is " + InetAddress.getLocalHost().getHostAddress();
    }
}

File: backend/src/main/kotlin/com/plugga/backend/TestController.kt
package com.plugga.backend

import java.net.InetAddress
import java.net.UnknownHostException
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class TestController {

    @GetMapping("/")
    @Throws(UnknownHostException::class)
    fun test(): String {
        return "Hello from TestController!!!! My address is " + InetAddress.getLocalHost().hostAddress
    }
}
